web-app/HeimScripts/heatmap/run.R

```diff
@@ -306,12 +290,12 @@ writeDataForZip <- function(df, zScores, pidCols) {
     df      <- df[ , -which(names(df) %in% pidCols)]  # Drop patient columns
     df      <- cbind(df,zScores)                      # Replace with zScores
     write.table(
-    df,
-    "heatmap_data.tsv",
-    sep = "\t",
-    na = "",
-    row.names = FALSE,
-    col.names = TRUE
+        df,
```

Comment:
I think all over the code we use 4 spaces. Why would we do it different in the R scripts?